lib/src/idx/trees/btree.rs

```diff
@@ -25,13 +26,13 @@ where
 }
 
 #[derive(Clone, Serialize, Deserialize)]
-#[revisioned(revision = 1)]
+#[revisioned(revision = 2)]
 pub struct BState {
 	minimum_degree: u32,
 	root: Option<NodeId>,
 	next_node_id: NodeId,
-	#[serde(skip)]
-	updated: bool,
+	#[revision(start = 2)]
+	generation: u64,
```

Comment:
Indexes have now a generation number. It is used by the index cache to know if the cache shold be invalided. Thanks to this mechanism, several SurealDB instances can be connected to an external KVStore and have the index cache be automatically flushed.